scoap3/modules/records/tasks.py

```diff
@@ -120,6 +122,12 @@ def performe_article_check_for_journal(from_date, journal, cooperation_dates):
             journal_stats['outside_cooperation_dates'] += 1
             continue
 
+        # if an article was published too recently, it shouldn't be reported
+        if pub_date >= ignore_after_datetime:
```

Comment:
only `>`, no?

Reply:
I'd ignore the articles for a bit longer in order not to have false negative alerts. In fact, the `pub_date` is midnight of the date of the publication, and `ignore_after_datetime` is 24 hours before 'now', so probably we should increase the difference for 48 hours. This way we will make sure we have only the really missing articles.